web/src/beta/components/SidePanelSectionField/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { useState, ReactNode } from "react";
+
+import { styled, useTheme, PublishTheme } from "@reearth/services/theme";
+
+import Icon from "../Icon";
+import Text from "../Text";
+
+const SidePanelSectionField: React.FC<{
+  publishedTheme?: PublishTheme;
+  title: string;
+  children?: ReactNode;
+}> = ({ publishedTheme, title, children }) => {
+  const theme = useTheme();
+  const [opened, setOpened] = useState(false);
+
+  return (
+    <Field>
+      <Header onClick={() => setOpened(!opened)}>
+        <Text size="xs" color={theme.other.white} otherProperties={{ height: "16px" }}>
+          {title}
+        </Text>
+        <Icon
+          icon="arrowToggle"
+          size={12}
+          color={publishedTheme?.mainText || theme.main.text}
+          style={{ rotate: opened ? "90deg" : "180deg" }}
```

Comment:
It is better to avoid using `style` prop as much as possible.
I guess you can add a kind of wrapper and use it instead of using `style` prop